css/index.css

```diff
@@ -0,0 +1,58 @@
+/*desktop style */
+body {
+    background-color: lightblue;
+    color: white;
+    font-family: Helvetica, "Times New Roman", sans-serif;
+            }
+
+
+  
+section{
+    margin: 10px;
+    padding: 20px;
+}
+
+#connect ul{
+    display: flex; 
+    flex-direction: row;
```

Comment:
As a quick tip, row is the default orientation of a flex direction. You can generally get away with not declaring this :)